packages/salesforcedx-slds-linter/test/server.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { validateTextDocument } from '../src/server/index';
+
+describe('SLDS Linter Language Server', () => {
+
+  let args: any;
+  const connection = {
+    sendDiagnostics: (obj: any) => {
+      args = obj;
+    }
+  };
+
+  it('Should send correct string', () => {
+
+    validateTextDocument('\n class"slds-button--brand" \n', 'uri', connection);
+    if (args) {
```

Comment:
@engai I think you should also assert that `args` is not null in the tests.
Because if it turns out that `args` is null, then if will skip the if(args) {...} and then the test will pass, right?

Reply:
Right. Pushed a fix to assert when null